src/types/UpdateCompleted.ts

```diff
@@ -0,0 +1,3 @@
+export interface UpdateCompleted {
+  completed: boolean,
+}
```

Comment:
Don't create for every type a file, if it's related to Todo put it in the same file